www/index.js

```diff
@@ -229,7 +247,7 @@ monaco.languages.onLanguage(modeId, async () => {
         }
     }
 
-    monaco.languages.setTokensProvider(modeId, {
+    /*monaco.languages.setTokensProvider(modeId, {
```

Comment:
Did you comment this out as you broke it, or for another reason?

Reply:
It is broken and for my experiment propose (adding to playground) a static highlighter was enough, so I removed it. But for "rust analyzer demo" we may want to bring it back even if it is broken and worse than static.

Break reproduce:
1. go to https://ra-wasm.netlify.app/
2. remove all the code
3. press ctrl+Z.